build/ci/create_env_file.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+
+import os
+
+
+def get_env_var(var):
+    try:
+        return os.environ[var]
+    except:
+        return ""
+
+
+def get_value_from_file(file):
+    try:
+        tf = open(file, "r")
```

Comment:
Might be better to use a `with` block here to close the file after reading it.

Reply:
Fixed